cmd/root.go

```diff
@@ -100,9 +100,9 @@ func WithDialer(d cloudsql.Dialer) Option {
 // NewCommand returns a Command object representing an invocation of the proxy.
 func NewCommand(opts ...Option) *Command {
 	cmd := &cobra.Command{
-		Use:     "cloud_sql_proxy instance_connection_name...",
+		Use:     "cloudsql-proxy instance_connection_name...",
```

Comment:
What if you use `filepath.Base(os.Args[0])` so that it will always be the executable filename? This way if we change the name of the executable, we won't break the help.

Reply:
That's an interesting idea. For now, I'd like to keep the binary as named after the module, and rename it all in one PR separate from this one.